### PR TITLE
rebase of the webcomponent from personal repo

### PR DESCRIPTION
**Pull request by cedricdcc**

### CODE REVIEW

File: public/code/person_id_tracker.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
+# python script to turn parentID into hierarchicall finesse
+# Made by Decruw Cedric and marc test
+
+# import 
+
+import csv
+import pandas as pd
+# dependencies
+
+path_file = "C:\\Users\\cedric\\Desktop\\new_excel_file_parentID.txt"
```
Comment (JotaFan) [inline public/code/person_id_tracker.py:10]: Hardcoded local files?

File: public/code/person_id_tracker.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+parentspcol = []
+with open(path_file, newline='',) as csvfile:
+    spamreader = csv.reader(csvfile, delimiter='\t', quotechar='|')
+    x = 0
+    for row in spamreader:
+        spcols.append(row[0])
+        parentspcol.append(row[3])
+        parentchilddic.append({'spcolid':row[0],'parent_spcolid':row[3]})
+        x+=1
+        #print(', '.join(row))
```
Comment (JotaFan) [inline public/code/person_id_tracker.py:35]: remove not used commented code

File: public/code/person_id_tracker.py [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    else:
+        hierarchy = [dic['spcolid']]
+        listhierarchies[x] = hierarchy
+    x+=1
+
+for key, value in listhierarchies.items():
+    print(value)
+
+        
+with open("C:\\Users\\cedric\\Desktop\\"+"new_parent_child_id.txt", 'w', newline='') as csvfilew:
```
Comment (JotaFan) [inline public/code/person_id_tracker.py:57]: again hardcoded local path.

Also paths should be done using `os.path` so it works on multiple OS

**Review by JotaFan — CHANGES_REQUESTED** [review]

File: public/code/person_id_tracker.py (hunk 1 above)
Comment (cedricdcc) [reply public/code/person_id_tracker.py:10]: Jup the best variables. This is also a file that is used for testing the preview functionality of the webcomponent.
But I agree that using local variables is a very bad practice.

File: public/code/person_id_tracker.py (hunk 2 above)
Comment (cedricdcc) [reply public/code/person_id_tracker.py:35]: This is also a file that is used for testing the preview functionality of the webcomponent.
I will solve this if the file is in any other repo.

File: public/code/person_id_tracker.py (hunk 3 above)
Comment (cedricdcc) [reply public/code/person_id_tracker.py:57]: This is also a file that is used for testing the preview functionality of the webcomponent.
But I agree that using local variables is a very bad practice.

**Review by JotaFan — APPROVED** [review]

**Pull request merged**